easybuild/easyconfigs/x/xcb-proto/xcb-proto-1.11.eb

```diff
@@ -0,0 +1,24 @@
+easyblock = 'ConfigureMake'
+
+name = 'xcb-proto'
+version = '1.11'
+
+homepage = 'http://xcb.freedesktop.org/'
+description = """The X protocol C-language Binding (XCB) is a replacement for Xlib featuring a small footprint,
+latency hiding, direct access to the protocol, improved threading support, and extensibility."""
+
+toolchain = {'name': 'dummy', 'version': 'dummy'}
```

Comment:
please add a comment above why it's OK to use `dummy` here (even though the easyblock we use if `ConfigureMake`)

```python
# even though xcb-proto is installed with configure-make-make install, nothing is actually built;
# only .py files are copied, so OK to use dummy toolchain
toolchain = ...
```